src/api/control/async/asyncLinearMotionProfileController.cpp

```diff
@@ -56,84 +56,46 @@ void AsyncLinearMotionProfileController::generatePath(std::initializer_list<QLen
     return;
   }
 
-  std::vector<Waypoint> points;
+  std::vector<squiggles::Pose> points;
   points.reserve(iwaypoints.size());
   for (auto &point : iwaypoints) {
-    points.push_back(Waypoint{point.convert(meter), 0, 0});
+    points.push_back(
+      squiggles::Pose{point.convert(meter), 0, 0});
```

Comment:
Thanks for catching this, I forgot about that detail. The Squiggles Pose is currently set up to be in cartesian coordinates rather than the ROS-style coordinate frame from Okapi and Pathfinder. I'll translate between these frames

Reply:
I spoke too soon, the coordinate frames should match between Squiggles and Pathfinder. Yaw is measured counterclockwise from the x-axis like Pathfinder.